src/main/resources/migration/changelog-v5.6.0.xml

```diff
@@ -33,4 +33,9 @@
         <modifyDataType tableName="VULNERABLESOFTWARE" columnName="UUID" newDataType="UUID"/>
         <modifyDataType tableName="WORKFLOW_STATE" columnName="TOKEN" newDataType="UUID"/>
     </changeSet>
+
+    <changeSet id="v5.6.0-2" author="rossmurphy974@gmail.com">
+        <renameColumn tableName="PROJECT" oldColumnName="AUTHOR" newColumnName="AUTHORS"/>
+        <renameColumn tableName="COMPONENT" oldColumnName="AUTHOR" newColumnName="AUTHORS"/>
```

Comment:
Renaming the column will break deserialization, since `AUTHORS` is expected to contain JSON, but `AUTHOR` was not a JSON field.

I think a better approach would be:

1. Create the new `AUTHORS` column
2. Populate `AUTHORS` based on values in the old `AUTHOR` column
3. Drop the `AUTHOR` column

For step (2), you could use the [`sql`](https://docs.liquibase.com/change-types/sql.html) change type like this (not tested):

```sql
UPDATE "PROJECT"
   SET "AUTHORS" = JSON_BUILD_ARRAY(JSON_BUILD_OBJECT('name', "AUTHOR"))::TEXT
 WHERE "AUTHOR" IS NOT NULL
```

See [here](https://www.postgresql.org/docs/12/functions-json.html) for details on the JSON functions. You can find other examples of the `sql` change types in changelog files for prior versions.